vector/src/main/java/im/vector/app/features/home/room/threads/list/viewmodel/ThreadListViewModel.kt

```diff
@@ -113,7 +113,15 @@ class ThreadListViewModel @AssistedInject constructor(@Assisted val initialState
 
     private fun fetchThreadList() {
         viewModelScope.launch {
+            isLoading(true)
             room?.fetchThreadSummaries()
+            isLoading(false)
+        }
+    }
+
+    private fun isLoading(show: Boolean) {
```

Comment:
I think `private fun setLoading(isLoading: Boolean)` is more convenient.

Reply:
Refactored!